Rename directory "scripts" to "tools"

takumiando opened the pull request:
"scripts" directory should be used for frozen modules.

ohsawa commented:
Great! Plz write a commit message about image size get smaller .

In my case, see below.
```
Memory region         Used Size  Region Size  %age Used
           FLASH:      369760 B       440 KB     82.07%
            SRAM:      118488 B       256 KB     45.20%
        IDT_LIST:         168 B         2 KB      8.20%
```

takumiando committed:
Rename directory "scripts" to "tools"

"scripts" directory should be used for frozen modules.

So after applying this patch, degu binary size gets smaller
about 433 KB to 370 KB.
